wherehows-web/tests/unit/utils/datasets/create-search-entries-test.js

```diff
@@ -14,7 +14,7 @@ module('Unit | Utility | datasets/create search entries', function(hooks) {
 
   test('it works base case', function(assert) {
     const { server } = this;
-    const dataset = server.create('dataset', 'forUnitTests');
+    const dataset = server.create('dataset', 'forUnitTests').attrs;
```

Comment:
Will there be a recollection to remember to remove this workaround when this [issue](https://github.com/samselikoff/ember-cli-mirage/issues/1379) is resolved

Reply:
I guess I need to add a comment here too.